src/components/shared/modal/modalContent/modalContent.scss

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+@import 'src/styles/mixins';
+@import 'src/styles/themes';
+
+.modal-content {
```

Comment:
These modal components are all used together. Use just one SCSS file (like Flyout does).

Reply:
That makes it tightly coupled. You can't reuse any sub-component independently. I've discussed with Stephen.

Reply:
I'm not sure that really applies here. You wouldn't use ModalContent outside of a modal... just like you wouldn't use FlyoutContent outside of a flyout. 
I'll defer to @spryor, but he himself set the precedent for this with the Flyout and other components he created.

Reply:
I agree you might not use ModalContent outside of a modal, but you might use a ModalFadebox outside of a modal.

Even if don't end up using it, I think it's cleaner to have separate .scss files.